src/application/downloader.py

```
@@ -0,0 +1,213 @@
+        self.data_source = data_source
+        self.storage = storage
+        self.job_manager = job_manager
+        self.metrics = metrics
+        self.max_concurrent_locations = max_concurrent_locations
+        self.max_concurrent_sensors = max_concurrent_sensors
+        self.metrics_middleware = MetricsMiddleware(metrics)
+
+    async def download_country(
+        self,
+        country_code: str,
+        parameters: Optional[List[ParameterType]] = None,
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)